### PR TITLE
contracts-bedrock: fix setup chain in tests

### PR DESCRIPTION
**Pull request by tynes**
**Description**
Fixes the foundry tests with the way that `setUp` is used, now its only used when necessary and removes `_setUp`

### CODE REVIEW

File: packages/contracts-bedrock/contracts/test/Semver.t.sol [hunk 1]
```diff
@@ -19,7 +19,8 @@ contract Semver_Test is CommonTest {
     /**
      * @notice Deploy a Semver contract
      */
-    function setUp() external {
+    function setUp() public virtual override {
+        super.setUp();
```
Comment (maurelian) [inline packages/contracts-bedrock/contracts/test/Semver.t.sol:23]: Seems we don't _need_ this, is it going to slow things down at all?

**Review by maurelian — APPROVED** [review]
1 comment, but LGTM

File: packages/contracts-bedrock/contracts/test/Semver.t.sol (hunk 1 above)
Comment (tynes) [reply packages/contracts-bedrock/contracts/test/Semver.t.sol:23]: There are definitely some that we don't need, happy to follow up with additional cleanup in the future

**Comment by tynes** [comment]
Ok @maurelian I fixed your issue and regenerated the snapshot

Overall these tests need better organization + cleanup

**Comment by tynes** [comment]
Perhaps we can slowly knock out cleaning up individual files

**Comment by maurelian** [comment]
Yeah, that's definitely the move. 

I would like to write issues so that the community can help out. This is good to go tho

**Pull request merged**